tests/base_scraper/test_correct_scraper.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import unittest
+import urllib
+
+import mozfile
+import mozhttpd_base_test as mhttpd
+from mozprocess import processhandler
+
+tests = [
+    # ReleaseScraper
+    {'command': ['-v', 'latest'],
+     'fname': 'firefox-latest.en-US.linux64.tar.bz2'},
+    # ReleaseCandidateScraper
```

Comment:
nit: lets leave blank lines between the examples so its easier to distinguish. I think we do the same for the other tests.